test/testusermodel.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ *    This software is in the public domain, furnished "as is", without technical
+ *    support, and with no warranty, express or implied, as to its usefulness for
+ *    any purpose.
+ *
+ */
+
+#include "localdiscoverytracker.h"
+#include "syncrunfilelog.h"
+#include "syncengine.h"
+#include <QtTest>
+
+#include "account.h"
+#define private public
```

Comment:
if I remember correctly, on windows `public` or `private` symbols in a shared library are encoded with different symbols
is this really needed and working ?

Reply:
Okay, sure.